buf.yaml

```diff
@@ -0,0 +1,25 @@
+version: v2
+modules:
+  - path: internal/proto
+  - path: internal/examples/pets/internal/proto
+    lint:
+      use:
+        - DEFAULT
+      except:
+        - ENUM_VALUE_PREFIX
+        - ENUM_VALUE_UPPER_SNAKE_CASE
+        - ENUM_ZERO_VALUE_SUFFIX
+        - RPC_REQUEST_RESPONSE_UNIQUE
+        - RPC_REQUEST_STANDARD_NAME
+        - RPC_RESPONSE_STANDARD_NAME
+deps:
+  - buf.build/googleapis/googleapis
+lint:
+  use:
+    - DEFAULT
+  except:
+    - RPC_REQUEST_RESPONSE_UNIQUE
+    - RPC_RESPONSE_STANDARD_NAME
```

Comment:
Should this also include `RPC_REQUEST_STANDARD_NAME`?

Reply:
Currently not needed, so removed to reduce excepts.